components/annotation_class.py

```diff
@@ -1,14 +1,17 @@
 import dash_mantine_components as dmc
 from dash import html, dcc
 from dash_iconify import DashIconify
+import uuid
+import dash_bootstrap_components as dbc
+import dash_daq as daq
```

Comment:
Can remove this? It's giving an error in the deploy since we don't have this in the `requirements.txt`

Reply:
oh daq yes! Ill remove that, but dbc is in req.txt